<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClearCase::Internal::SettingsPage</class>
 <widget class="QWidget" name="ClearCase::Internal::SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>589</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="generalGroupBox">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="commandLabel">
        <property name="text">
         <string>&amp;Command:</string>
        </property>
        <property name="buddy">
         <cstring>commandPathChooser</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::PathChooser" name="commandPathChooser" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Diff</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="graphicalDiffRadioButton">
        <property name="text">
         <string>&amp;Graphical (single file only)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="externalDiffRadioButton">
        <property name="text">
         <string>&amp;External</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="diffWidget" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="diffArgsLabel">
           <property name="text">
            <string>Arg&amp;uments:</string>
           </property>
           <property name="buddy">
            <cstring>diffArgsEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="diffArgsEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="diffWarningLabel">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>68</red>
              <green>96</green>
              <blue>92</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="miscGroupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="historyCountLabel">
        <property name="text">
         <string>&amp;History count:</string>
        </property>
        <property name="buddy">
         <cstring>historyCountSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="historyCountSpinBox">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="timeOutLabel">
        <property name="text">
         <string>&amp;Timeout:</string>
        </property>
        <property name="buddy">
         <cstring>timeOutSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="timeOutSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="autoCheckOutCheckBox">
        <property name="text">
         <string>&amp;Automatically check out files on edit</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="promptCheckBox">
        <property name="text">
         <string>&amp;Prompt on check-in</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="disableIndexerCheckBox">
        <property name="text">
         <string>Di&amp;sable indexer</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="indexOnlyVOBsLabel">
        <property name="text">
         <string extracomment="VOB: Versioned Object Base">&amp;Index only VOBs:</string>
        </property>
        <property name="buddy">
         <cstring>indexOnlyVOBsEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="indexOnlyVOBsEdit">
        <property name="toolTip">
         <string>VOBs list, separated by comma. Indexer will only traverse the specified VOBs. If left blank, all active VOBs will be indexed.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="autoAssignActivityCheckBox">
        <property name="toolTip">
         <string>Check this if you have a trigger that renames the activity automatically. You will not be prompted for activity name.</string>
        </property>
        <property name="text">
         <string>Aut&amp;o assign activity names</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="noCommentCheckBox">
        <property name="toolTip">
         <string>Check out or check in files with no comment (-nc/omment).</string>
        </property>
        <property name="text">
         <string>Do &amp;not prompt for comment during checkout or check-in</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>graphicalDiffRadioButton</tabstop>
  <tabstop>externalDiffRadioButton</tabstop>
  <tabstop>diffArgsEdit</tabstop>
  <tabstop>historyCountSpinBox</tabstop>
  <tabstop>timeOutSpinBox</tabstop>
  <tabstop>autoCheckOutCheckBox</tabstop>
  <tabstop>autoAssignActivityCheckBox</tabstop>
  <tabstop>noCommentCheckBox</tabstop>
  <tabstop>promptCheckBox</tabstop>
  <tabstop>disableIndexerCheckBox</tabstop>
  <tabstop>indexOnlyVOBsEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>externalDiffRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>diffWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
